Documentation/operations/upgrade.rst

```diff
@@ -346,10 +329,22 @@ Removed Options
   ``disable-conntrack`` option and made it non-operational. It will be removed
   in version 1.13.
 
-Deprecated Options
-~~~~~~~~~~~~~~~~~~
+Helm Options
+~~~~~~~~~~~~
 
-* ``bpf.hostRouting`` has been deprecated in favor of ``bpf.hostLegacyRouting``, and will be removed in 1.13.
+* ``bpf.hostRouting`` has been deprecated in favor of ``bpf.hostLegacyRouting``, and
+    will be removed in 1.13.
```

Comment:
Formatting is currently broken, see below from the preview. I think this is because you need to align the continuation lines with the beginning of the first line, see example below:

```suggestion
* ``bpf.hostRouting`` has been deprecated in favor of ``bpf.hostLegacyRouting``, and
  will be removed in 1.13.
```

![image](https://user-images.githubusercontent.com/17001771/157423654-5eea8102-de9b-4762-8b6d-38de4e4fa536.png)

Reply:
thanks, I have verified again with make render-docs-live-preview this time :+1: